hackathon_site/dashboard/frontend/src/components/inventory/InventoryFilter/InventoryFilter.test.tsx

```diff
@@ -72,19 +82,21 @@ describe("<InventoryFilter />", () => {
     });
 
     it("Clears filters when toggled", async () => {
-        const { getByText } = render(<InventoryFilter />);
+        const store = prepopulateStore();
+        const { getByText, findByText } = render(<InventoryFilter />, { store });
 
         const orderByNameButton = getByText("Z-A");
         const inStockCheckbox = getByText("In stock");
-        const FPGACheckbox = getByText("FPGA");
+        const firstCheckbox = await findByText(mockCategories[0].name);
```

Comment:
Calling this out for anyone who's curious - `getByText()` will error immediately if the text isn't there, `findByText` will return a promise (with I think a 5 second timeout) for the data to appear.

Since it takes a bit of time for the action dispatched in `prepopulateStore()` to make it's way through, hit the (mocked)  API, etc, this is just waiting for the categories to be available in the store / filters.